m4/ax_check_openssl.m4

```diff
@@ -39,7 +39,7 @@ AC_DEFUN([AX_CHECK_OPENSSL], [
     found=false
     AC_ARG_WITH([openssl],
         [AS_HELP_STRING([--with-openssl=DIR],
-            [root of the OpenSSL directory])],
+            [override root of the OpenSSL directory to DIR])],
```

Comment:
Changing this file doesn't seem to implicate the license

Reply:
I'm not sure what this means, sorry.